dll/win32/syssetup/wizard.c

```diff
@@ -2992,6 +3020,12 @@ ProcessSetupInf(
                              (LPBYTE)pSetupData->SourcePath,
                              (wcslen(pSetupData->SourcePath) + 1) * sizeof(WCHAR));
 
+        res = RegSetValueExW(hKey,
+                             L"Installation Sources",
+                             0,
+                             REG_MULTI_SZ,
+                             (LPBYTE)szValue,
+                             (DWORD)((pData - szValue + 1) * sizeof(WCHAR)));
```

Comment:
This actually introduces a problem. If there are existing values in `Installation Sources`, they will be just dropped and rewritten by two added lines.

I'm not sure we want this behavior. It should probably add new lines to that value, while checking if they are already added.

Also it seems `newdev.dll` handles updating this value on its own: https://git.reactos.org/?p=reactos.git;a=blob;f=dll/win32/newdev/wizard.c;hb=79b0fce5dcfd11d1ecb6880929d3660a17b34b45#l358

Reply:
That's a good point.

Reply:
I have added code that resolves this problem.